ui/page/components/seed_restore_page.go

```diff
@@ -518,6 +470,8 @@ func (pg *SeedRestore) verifySeeds() bool {
 }
 
 func (pg *SeedRestore) resetSeeds() {
+	pg.allSuggestions = pg.getWordSeedType().AllSeeds()
```

Comment:
What does `AllSeed` return? Words right? I think it can use a rename -> `AllWords`?